packages/atomic/src/themes/coveo.css

```diff
@@ -22,6 +22,13 @@
   --atomic-success-background: #d4fcf0;
   --atomic-error-background: #fcbdc0;
   --atomic-primary-background: #edf6ff;
+  --atomic-inline-code: #cd2113;
+
+  /* Table colors */
+  --atomic-table-outer-border: #bfc4c8;
```

Comment:
I get why you added those, but it means that other tables used in Atomic should be retrofitted to start using those variable.

Otherwise this is causing theming/inconsistency issues for implementers

There's some mixin that were created for `atomic-result-table`. I wonder if they could be genericized/extracted and reused for this use case... 🤔 

In `components/common/item-list/styles/mixins.pcss`

Reply:
Thanks for the pointer, I'll look to harmonize that.

Reply:
I can't get close enough to the target look and feel using the existing colors. Is it ok with you if I add one more shade of gray between `--atomic-neutral` and `--atomic-neutral-dark`? Not sure which name to use though...

Reply:
@olamothe I went for `--atomic-neutral-dim` which, I think can fit between `--atomic-neutral` and `--atomic-neutral-dark`. Let me know if it makes sense to you.